utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
```

Comment:
We should add a top-level documentation comment to this file explaining what sorts of files belong to it (e.g. ``General purposes utilities pertaining to the Android accessibility manager.``).

Reply:
Done